### DISCUSSION TITLE
hydra, kali-linux and hyper-v

### DISCUSSION THREAD

**Discussion by Led1403** [ory/hydra#3647] [Q&A] [answered]
my Instagram account was lost and there are some people pretending to be me and talking to my friends as if they were me. With my knowledge of Hydra, Kali-Linux and Hyper-V I tried to find a code to force try all possible password combinations, but it's not working, can anyone help?

**Comment by vinckr** [comment, marked as answer]
Hello @Led1403 
I think you are looking for a different Hydra: https://github.com/vanhauser-thc/thc-hydra
